dfx.json

```diff
@@ -51,7 +51,9 @@
 			"shrink": false,
 			"remote": {
 				"id": {
-					"staging": "qbw6f-caaaa-aaaah-qdcwa-cai"
+					"staging": "qbw6f-caaaa-aaaah-qdcwa-cai",
+					"beta": "qgxyr-pyaaa-aaaah-qdcwq-cai",
```

Comment:
Do we need this? Why isn't it needed in the other canisters?

Do we also need to support this env in `deploy.backend.sh`?

Reply:
According your code we do need canister Ids for "beta" and "ic" to use "https://id.decideai.xyz" so I guess yes there are needed.